package.json

```diff
@@ -4,11 +4,16 @@
   "description": "Flux Standard Action utlities for Redux",
   "main": "lib/index.js",
   "scripts": {
-    "test": "make test",
-    "prepublish": "make clean build"
+    "build": "npm run clean && babel src --out-dir lib --ignore *-test.js",
```

Comment:
I like the idea of `import 'chai'` - no need for extra build complexity just to set up that one global. It would also help clarify what assertion library we're using.